pmm-app/src/pmm-qan/panel/components/Details/Example/ParseError/ParseError.tsx

```diff
@@ -0,0 +1,21 @@
+import { useStyles } from '@grafana/ui';
+import React from 'react';
+import { MAX_QUERY_LENGTH_DOCS_LINK } from './ParseError.constants';
+import { Messages } from './ParseError.messages';
+import { getStyles } from './ParseError.styles';
+
+const ParseError: React.FC = () => {
+  const styles = useStyles(getStyles);
+
+  return (
+    <pre data-testid="example-query-invalid">
+      {Messages.start}
+      <a className={styles.link} href={MAX_QUERY_LENGTH_DOCS_LINK} target="_blank" rel="noreferrer">
```

Comment:
```suggestion
      <a className={styles.link} href={MAX_QUERY_LENGTH_DOCS_LINK} target="_blank" rel="noreferrer noopener">
```

Reply:
Link to docs will be added a later date because of time constrains.